c172-sound.xml

```diff
@@ -149,7 +152,6 @@
         
         <gyro>
             <name>gyro</name>
-            <type>avionics</type>
```

Comment:
Why did you remove `<type>` here? AFAIK it's used so it's volume can be controlled in FG's sound settings dialog.

Reply:
Okay, you've already answered this in the issue.

Reply:
I did that because if a sound is of type <type>avionics</type> then it is not affected by the max-distance for some unknown reason to me. It was the only way I could get this to work... see https://github.com/c172p-team/c172p-detailed/issues/1015#issuecomment-373998045

Reply:
I think it's a small price to pay in order to have the sound volume working in all views, which is more important to me. What do you think?

Reply:
:+1: